phpBB/adm/style/acp_ext_list.html

```diff
@@ -42,59 +42,50 @@ <h1>{L_EXTENSIONS_ADMIN}</h1>
 		</tr>
 	</thead>
 	<tbody>
-		<!-- IF .enabled -->
-		<tr>
+		
+		<tr id="ext_enabled_header"<!-- IF not .enabled --> class="hidden"<!-- ENDIF -->>  
 			<td class="row3" colspan="4"><strong>{L_EXTENSIONS_ENABLED}</strong><!-- EVENT acp_ext_list_enabled_title_after --></td>
 		</tr>
-		<!-- BEGIN enabled -->
+		{% for ext in enabled %}
```

Comment:
I don't think it's a good idea to have mixed template syntaxes. We should probably just stick with old syntax, ugly as it is. Also it looks like you're just using twig to rename the loop array fron `enabled` to `ext` so I don't think this may be necessary.

Reply:
Yes but that allowed me to use only one loop in `acp_ext_actions.html`. Now it needs 2 loops.